### PR TITLE
FIx RadioButton cause RadioGroup onChange event trigger twice

### LINKED ISSUE #14485: RadioGroup onChange事件执行了两次
antd 版本 3.12.4

问题：
今日更新npm库时，更新了antd。原功能是RadioGroup变更时，去请求数据一次，今日发现，请求重复了。由RadioGroup的onChange事件执行了两次引起的。

代码如下

`
<RadioGroup
        onChange={(e) => {
          e.preventDefault();
          // 查询数据
          query({
            isMy: e.target.value
          });
        }}
      >
        <RadioButton value={false}>
          全部
        </RadioButton>
        <RadioButton value>
          我的
        </RadioButton>
      </RadioGroup>
`
---

### PR DESCRIPTION
**Pull request by ztplz**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
Fix https://github.com/ant-design/ant-design/issues/14485
> 3. Related issue link.
https://github.com/ant-design/ant-design/issues/14485
  
### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
Fix the problem that if the RadioButton exists, it will cause the RadioGroup onChange callback to trigger twice.
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/radio/radio.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class Radio extends React.Component<RadioProps, {}> {
   };
 
   onChange = (e: RadioChangeEvent) => {
-    if (this.props.onChange) {
+    if (this.props.onChange && this.props.prefixCls !== 'ant-radio-button') {
```
Comment (zombieJ) [inline components/radio/radio.tsx]: 这不行的，用户如果用 configProvider 改了 prefix 就不对了……

Comment (ztplz) [reply components/radio/radio.tsx]: 那就个radioButtonOnChange 和 radio的 onChange区分开

Comment (zombieJ) [reply components/radio/radio.tsx]: 可以直接移除掉 RadioButton 里的 onChange 不？

Comment (ztplz) [reply components/radio/radio.tsx]: 可以，button没有自己的onchange事件，原来是直接把group的onchange函数直接传下去的

**Comment by ztplz** [comment]
Done.

**Comment by zombieJ** [comment]
加一个 test case 记一下数哈~

**Review by afc163 — APPROVED** [review]

**Pull request merged**